extension/styles.css

```diff
@@ -0,0 +1,12 @@
+body {
```

Comment:
Doesn't adding classes bulk down the code? Haven't heard this before, I will have to look into this convention. Thank you.